nomad/conf/validators.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+import re
+
+from voluptuous.schema_builder import message
+from voluptuous.validators import truth
+
+
+hostname_part_re = re.compile(r'(?!-)[A-Z\d-]{1,63}(?<!-)$', re.IGNORECASE)
+lxd_identifier_re = re.compile(r'^(([a-zA-Z]|[a-zA-Z][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z]|[A-Za-z][A-Za-z0-9\-]*[A-Za-z0-9])$', re.IGNORECASE)  # noqa
```

Comment:
Since you have that `IGNORECASE` flag, can't we do like in `hostname_part_re` and use `A-Z` only?